docs/content/_index.md

```diff
@@ -1002,7 +1002,7 @@ opa run example.rego repl.input:input.json
 ```
 
 ```ruby
-> data.example.public_server[s]
+> data.example.public_servers[s]
```

Comment:
The rule is defined as `public_server` here, though: https://github.com/open-policy-agent/opa/pull/6491/files#diff-5d9ba8bc1a568741b44ccc852748ca739d71f89dc1ac05a9f8d3787a064a549dR907

So either we keep it as-is, or change both locations (and perhaps others, if there are any).